src/imageEditor.rei

```diff
@@ -0,0 +1,73 @@
+type imageSource = [ | `URI(string) | `Required(Packager.required)];
+
+type offset = {
+  x: int,
+  y: int,
+};
+
+type size = {
+  width: int,
+  height: int,
+};
+
+[@bs.deriving jsConverter]
+type resizeModeT = [
+  | [@bs.as "contain"] `Contain
+  | [@bs.as "cover"] `Cover
+  | [@bs.as "stretch"] `Stretch
+];
```

Comment:
Needs a change, see 
https://github.com/reasonml-community/bs-react-native/pull/280/files#r256710619

Reply:
Could you clarify please? Changes in line with what I've proposed (removing `bs.as` annotations, etc.) or anything beyond that?

Reply:
I guess anything that will makes compiler happy? :)
I am still new to this bs.* directives

Reply:
Alright then, I was confused because with my changes in place, I get no errors or warnings from the compiler and the module seems to work as expected in the app.